src/SFA.DAS.EmployerIncentives.Abstractions/DTOs/ApprenticeshipDto.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace SFA.DAS.EmployerIncentives.Abstractions.DTOs
+{
+    public class ApprenticeshipDto
+    {
+        public long UniqueLearnerNumber { get; set; }
+        public DateTime StartDate { get; set; }
+        public bool IsApproved { get; set; }
```

Comment:
Not quite sure what the intent of the field IsApproved. Would it be better to say IsEligible